docs/source/advanced.rst

```diff
@@ -29,35 +29,40 @@ By default, all OnionShare services are protected with a private key, which Tor
 
 When browsing to an OnionShare service in Tor Browser, Tor Browser will prompt for the private key to be entered.
 
-Sometimes you might want your OnionShare service to be accessible to the public, like if you want to set up an OnionShare receive service so the public can securely and anonymously send you files.
+You can make a OnionShare service accessible to the public. Set up an OnionShare receive service so the public can securely and anonymously send you files.
```

Comment:
The receive mode service is just a single example of when you might want to make a service public. Instead of, "Set up an OnionShare receive service", it should be something like, "For example, you can set up an OnionShare receive service"